libdnf/repo/solv_repo.cpp

```diff
@@ -277,6 +278,49 @@ void SolvRepo::load_repo_main(const std::string & repomd_fn, const std::string &
 }
 
 
+void SolvRepo::load_system_repo_ext(RepodataType type) {
+    auto & logger = *base->get_logger();
+    auto & pool = get_pool(base);
+    int solvables_start = pool->nsolvables;
+    auto type_name = repodata_type_to_name(type);
+    int res = 0;
+    switch (type) {
+        case RepodataType::COMPS: {
+            // get installed groups from system state and load respective xml files
+            // to the libsolv pool
+            auto & system_state = base->p_impl->get_system_state();
+            auto comps_dir = system_state.get_group_xml_dir();
+            auto installed_groups = system_state.installed_groups();
+            for (const auto & group_id : installed_groups) {
+                auto ext_fn = comps_dir / (group_id + ".xml");
+                fs::File ext_file;
+                try {
+                    ext_file = fs::File(ext_fn, "r", true);
+                } catch (std::filesystem::filesystem_error & e) {
+                    logger.debug(
```

Comment:
Is this really a "debug" message? Is it logging something that is standard behavior?  It looks more like a warning. We want to warn the user that a loading error has occurred.